libnopegl/src/nodes.c

```diff
@@ -291,21 +291,6 @@ static int node_init(struct ngl_node *node)
     return 0;
 }
 
-static int node_set_ctx(struct ngl_node *node, struct ngl_ctx *ctx, struct ngl_ctx *pctx);
```

Comment:
nit: ngli_node_children_apply_func() in commit msg

Reply:
Done in 2-3 other commits